source/retryable-reads/tests/README.rst

```diff
@@ -148,16 +148,17 @@ data.
 .. _GridFSBucket spec: https://github.com/mongodb/specifications/blob/master/source/gridfs/gridfs-spec.rst#configurable-gridfsbucket-class
     
 Speeding Up Tests
-===================
+-----------------
 
 Drivers may benefit reducing `minHeartbeatFrequencyMS`_ in order to speed up
 tests. Python was able to decrease the run time of the tests greatly by lowering
 the SDAM's ``minHeartbeatFrequencyMS`` from 500ms to 50ms, thus decreasing the
 waiting time after a "not master" error:
 
-.. _minHeartbeatFrequencyMS: https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#minheartbeatfrequencyms
+Also, similar improvement can be added for `heartbeatFrequencyMS`_ by setting the default heartbeatFrequencyMS to 5ms in order to take into account the latest changes regarding streaming protocol. If a test has an explicit heartbeatFrequencyMS value, drivers should use the explicit value.
```

Comment:
The original paragraph still states 50ms for `minHeartbeatFrequencyMS`, which contradicts 5ms here.

I propose consolidating both paragraphs into one:

```
Drivers can greatly reduce the execution time of tests by setting `heartbeatFrequencyMS`_
and `minHeartbeatFrequencyMS`_ (internally) to a small value (e.g. 5ms), below what
is normally permitted in the SDAM spec. If a test specifies an explicit value for
heartbeatFrequencyMS (e.g. client or URI options), drivers SHOULD use that value.
```

Reply:
I don't think the anecdote about the Python driver using 50ms is particular relevant anymore, so I have no qualms about removing that. Also, I think "(internally)" is sufficient to explain that we're doing something that wouldn't be permitted by a driver's public API.

cc: @ShaneHarvey

Reply:
sgtm, done